### DISCUSSION TITLE
Swap compat with Wasm variant in examples

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#119] [General]
It looks like all of the examples are using a compat variant of the build. I think it should be used as a fallback for users, who are not able to use Wasm variant for some reason. It is also larger in size due to embedding stuff, which makes it harder to use in environments, where size matters (e.g. Facebook Instant Games, where everything must be in a single archive, no external links and there is a size limit on the archive). Examples using a compat variant suggest that it is a preferred way of instantiating Jolt. It is a trivial change, but wanted to check if there was something else that moved the scales towards compat use.

```js
// In case you haven't built the library yourself, replace URL with: 
// https://www.unpkg.com/jolt-physics/dist/jolt-physics.wasm.js
import initJolt from './../js/jolt-physics.wasm.js';

const Jolt = await initJolt();

// Initialize this example
initExample(Jolt, null);
...
```

**Comment by jrouwe** [comment]
TBH I don't know if there are any downsides to using the wasm version. Maybe there are browsers that fail with the wasm version and not with the compat version? Both versions are provided in the bundle, so everyone can pick the one they like best. From a performance standpoint it shouldn't matter.
